diff -urN /tmp/venndor-diff-3799486661/is-odd/index.js vendor/is-odd/index.js
--- /tmp/venndor-diff-3799486661/is-odd/index.js	2026-07-23 14:29:59.305121168 +0100
+++ vendor/is-odd/index.js	2026-07-23 14:29:59.011794256 +0100
@@ -20,6 +20,6 @@
   if (!Number.isSafeInteger(n)) {
     throw new Error('value exceeds maximum safe integer');
   }
-  return (n % 2) === 1;
+  return (n % 2) !== 0;
 };
 
